Deep_learning/Deep-learning.rst

```diff
@@ -274,6 +274,10 @@ This is a quick guide to setup PyTorch with ROCm support inside a docker contain
 
 Note: Currently, ROCm install version 3.3 is required.
 
+
+*********
+Option 1
```

Comment:
Formating of 'option 1', 'option 2', 'option 3' must be same ? Add title for option 1.